### PR TITLE
Add "Ricochet Refresh.app" v3.0.10

### PR DESCRIPTION
**Pull request by m-simonelli**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

Hi
We're looking to get [Ricochet Refresh](ricochetrefresh.net) integrated into homebrew.
Fixed, see comment. ~~One issue was encountered when running `brew audit --cask ricochet-refresh`, see output:~~
```
noone@noone homebrew-cask % brew audit --new-cask ricochet-refresh
audit for ricochet-refresh: failed
 - exception while auditing ricochet-refresh: undefined method `downcase' for nil:NilClass
Error: 1 problem in 1 cask detected
noone@noone homebrew-cask % uname -a
Darwin noone.modem 20.3.0 Darwin Kernel Version 20.3.0: Thu Jan 21 00:06:51 PST 2021; root:xnu-7195.81.3~1/RELEASE_ARM64_T8101 x86_64
noone@noone homebrew-cask %
```
~~Not sure if this is an error on my part or if it's a bug in `brew audit` (I tried to follow the docs and keep it fairly simple, but granted this is my first time packaging for homebrew).~~

Cheers,
M

### CODE REVIEW

**Comment by m-simonelli** [comment]
Update:
`brew audit` runs successfully - Just had to add a `https://www.` to the start of the homepage link

Cheers,
M

**Comment by gdams** [comment]
Looking at the [README.md](https://github.com/blueprint-freespeech/ricochet-refresh/#readme) it clearly states "WORK IN PROGRESS PROJECT REPOSITORY - NOT FOR PRODUCTION USE".

I also note that the GitHub repository is a fork of https://github.com/ricochet-im/ricochet?

**Comment by m-simonelli** [comment]
cc @pospeselr we need to fix this
@gdams regarding both points, see https://github.com/ricochet-im/ricochet/issues/613
TLDR Ricochet, the original project, is essentially dead so we at blueprint forked the code to "refresh" it, hence "ricochet refresh" was born.
We're stable and have a release on [the website](https://ricochetrefresh.net) - the readme note is more a of a [documentation lagging behind reality as usual](https://github.com/blueprint-freespeech/ricochet-refresh/issues/121)

Cheers,
M

**Comment by m-simonelli** [comment]
Update: [see tentative update README](https://github.com/blueprint-freespeech/ricochet-refresh/pull/125)

**Comment by gdams** [comment]
>TLDR Ricochet, the original project, is essentially dead so we at blueprint forked the code to "refresh" it, hence "ricochet refresh" was born.

If the original repo is abandoned and upstream is therefore never going to merge back it might be worth un-forking that repository.

**Comment by m-simonelli** [comment]
> > TLDR Ricochet, the original project, is essentially dead so we at blueprint forked the code to "refresh" it, hence "ricochet refresh" was born.
> 
> If the original repo is abandoned and upstream is therefore never going to merge back it might be worth un-forking that repository?

Not really in my control, unfortunately, and does it particularly matter? It signals that this project is in fact a child project of the original ricochet (I mean we even gave it a different name, it's not like we're trying to compete with a project of the same name)

Cheers,
M

**Comment by miccal** [comment]
Looks fine to me, thank you @m-simonelli.
